src/Tizen.Sensor/Tizen.Sensor/SensorEnumerations.cs

```diff
@@ -319,4 +323,41 @@ public enum DetectorState
         /// <since_tizen> 3 </since_tizen>
         Detected = 1
     }
+    
+    /// <summary>
+    /// The auto rotation state.
+    /// </summary>
+    /// <since_tizen> 3 </since_tizen>
+    public enum AutoRotationState
+    {
+        /// <summary>
+        /// Unknown.
+        /// </summary>
+        /// <since_tizen> 3 </since_tizen>
+        Unknown = -1,
```

Comment:
I recommend to remove Unknown value. 
you do better throw exception rather than return unknown value
or use Degree_0 as default value

Reply:
@myroot  done! :smile: